examples/cluster/plot_cluster_comparison.py

```diff
@@ -27,7 +27,7 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
-from sklearn import cluster, datasets
+from sklearn import cluster, datasets, mixture
 from sklearn.neighbors import kneighbors_graph
 from sklearn.preprocessing import StandardScaler
 
```

Comment:
I am happy to do this, but all of the other `plot_*.py` examples have this line before the imports.  Maybe it is so you can see progress when you `make` the docs?  

Any other opinions on whether this should be removed?